app/src/main/java/com/example/sharingang/users/FirestoreUserStore.kt

```diff
@@ -30,4 +35,22 @@ class FirestoreUserStore @Inject constructor(firestore: FirebaseFirestore) :
         return super.update(user, user.id)
     }
 
+    override suspend fun report(reportedUser: User, reporterUser: User): Boolean {
+        return try {
+            firestore
+                .collection("users").document(reportedUser.id!!)
+                .collection("reports").document(reporterUser.id!!).set(
+                    hashMapOf(
+                        reporterUser.id to Calendar.getInstance().time.toString()
+                    )
+                )
+            Log.d("TESTING FIRE", "User ${reporterUser.id} reported ${reportedUser.id}")
```

Comment:
```suggestion
            Log.d(TAG, "User ${reporterUser.id} reported ${reportedUser.id}")
```

Reply:
Oh my I completely forgot about that! Thanks!